BaseClasses.py

```diff
@@ -400,7 +455,17 @@ def get_game_worlds(self, game_name: str):
                      player not in self.groups and self.game[player] == game_name)
 
     def get_name_string_for_object(self, obj: HasNameAndPlayer) -> str:
-        return obj.name if self.players == 1 else f'{obj.name} ({self.get_player_name(obj.player)})'
+        if self.players == 1:
+            return obj.name
+
+        player = obj.player
+        if player in self.groups:
+            # Include all the players in the group.
+            group_players = self.groups[player]["players"]
```

Comment:
`group_players` is a `set`, so has no particular ordering. It might be beneficial if the players were sorted into a specific order. Sorting `group_players` within each `get_name_string_for_object` call does not seem like it would be a good idea, but maybe a `"sorted_players"` key could be added to `Group` that will contain the players after sorting.